templates/registration/login.html

```diff
@@ -19,16 +20,21 @@ <h6 class="text-sm font-bold text-blueGray-500">
                     </h6>
                 </div>
                 <div class="text-center btn-wrapper">
+                    
+                    
+                    <a href="{% provider_login_url 'github' %}?next=/">
```

Comment:
```suggestion
                    <a href="{% provider_login_url 'github' %}?next=/">
```
在進入 `login` 頁面，這裡好像會壞掉